tests/python/unittest/test_operator.py

```diff
@@ -1564,6 +1564,13 @@ def test_bmin(a, b):
         # pass idx=200 to gen_broadcast_data so that generated ndarrays' sizes are not too big
         data = gen_broadcast_data(idx=200)
         check_bmaxmin_gradient(c, data[0], data[1], 0.001, 1e-2, 1e-3)
+        
+    def test_band(a, b):
+        c = mx.sym.broadcast_logical_and(a, b)
+        check_binary_op_forward(c, lambda x, y: np.logical_and(x, y), gen_broadcast_data, mx_nd_func=mx.nd.logical_and)
+        # pass idx=200 to gen_broadcast_data so that generated ndarrays' sizes are not too big
+        data = gen_broadcast_data(idx=200)
+        check_bmaxmin_gradient(c, data[0], data[1], 0.001, 1e-2, 1e-3)
```

Comment:
Is this a proper gradient check for this op?

Reply:
no its not. let me push the changes.